airbyte-integrations/connectors/source-monday/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 80a54ea2-9959-4040-aac1-eee42423ec9b
-  dockerImageTag: 2.0.4
+  dockerImageTag: 2.0.5
```

Comment:
This should be a minor version change per Brian's comment [here](https://github.com/airbytehq/airbyte/pull/36267#pullrequestreview-1950124151)

Reply:
Thanks
updated